lua/astrocommunity/pack/java/init.lua

```diff
@@ -19,7 +19,7 @@ return {
     "jay-babu/mason-null-ls.nvim",
     optional = true,
     opts = function(_, opts)
-      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "clang_format" })
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "clang-format" })
```

Comment:
The null-ls formatter is called clang_format. When using `:NullLsInstall` it appears as `clang_format` in the list.

Reply:
ah ic, i somehow had it in my memory that it didnt worked on my work pc and had to fix it. ig i forgot to update then, sry to bother